<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageAccountWidget</class>
 <widget class="QWidget" name="ManageAccountWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QTableView" name="accountTableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>480</width>
     <height>230</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="selectionLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>471</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
